From f648c971c914bac00b20d31b6a170bf5b55f7027 Mon Sep 17 00:00:00 2001
From: Florian Dollinger <dollinger.florian@gmx.de>
Date: Mon, 8 Jan 2018 12:53:16 +0100
Subject: [PATCH] Added hid-xpadneo to Kconfig (hid)

Signed-off-by: Florian Dollinger <dollinger.florian@gmx.de>
---
 drivers/hid/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/hid/Kconfig b/drivers/hid/Kconfig
index 779c5ae47..50b6edae9 100644
--- a/drivers/hid/Kconfig
+++ b/drivers/hid/Kconfig
@@ -969,6 +969,13 @@ config HID_XINMO
 	standard. Currently only supports the Xin-Mo Dual Arcade. Say Y here
 	if you have a Xin-Mo Dual Arcade controller.
 
+config HID_XPADNEO
+	tristate "XBOX ONE S and X controller over Bluetooth, including FF"
+	depends on HID
+	select INPUT_FF_MEMLESS
+	---help---
+	Support for Xbox One S and X controllers over BT, including force-feedback.
+
 config HID_ZEROPLUS
 	tristate "Zeroplus based game controller support"
 	depends on HID
-- 
2.15.1

