From 708657ad586543fb877a46e6452511973d92a00a Mon Sep 17 00:00:00 2001
From: Florian Dollinger <dollinger.florian@gmx.de>
Date: Mon, 8 Jan 2018 12:45:34 +0100
Subject: [PATCH] Added hid-xpadneo to Makefile

Signed-off-by: Florian Dollinger <dollinger.florian@gmx.de>
---
 drivers/hid/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hid/Makefile b/drivers/hid/Makefile
index 235bd2a7b..57824e588 100644
--- a/drivers/hid/Makefile
+++ b/drivers/hid/Makefile
@@ -99,3 +99,4 @@ obj-$(CONFIG_HID_UCLOGIC)	+= hid-uclogic.o
 obj-$(CONFIG_HID_LED)		+= hid-led.o
 obj-$(CONFIG_HID_XINMO)		+= hid-xinmo.o
+obj-$(CONFIG_HID_XPADNEO)	+= hid-xpadneo.o
 obj-$(CONFIG_HID_ZEROPLUS)	+= hid-zpff.o
 
-- 
2.15.1

